composables/useRmrkVersion.ts

```diff
@@ -3,14 +3,14 @@ import { RemarkVersion } from '@kodadot1/minimark/shared'
 export default function () {
   const { urlPrefix } = usePrefix()
 
-  const version = computed<RemarkVersion | null>(() => {
+  const version = computed<RemarkVersion>(() => {
     switch (urlPrefix.value) {
       case 'rmrk':
         return '1.0.0'
       case 'ksm':
         return '2.0.0'
       default:
-        return null
+        return '2.0.0'
```

Comment:
why?

Reply:
I originally wanted to keep the same implementation as on the mixin, but now I think it's not that suitable.